script/install_sara_packages.bash

```diff
@@ -16,4 +16,13 @@ rosdep install --from-paths src --ignore-src --rosdistro kinetic -y
 
 catkin_make
 
+
+# Create source entry in bashrc
+echo "source ~/sara_ws/devel/setup.bash" >> ~/.bashrc
+
+# Add ROS ip adress for communication with non local nodes
+# Need to update IP adress in function of the network configuration
+echo "export ROS_IP=192.168.0.118" >> ~/.bashrc
```

Comment:
change to echo "#export ROS_IP=192.168.0.118">> ~/.bashrc

We might change the ip address later